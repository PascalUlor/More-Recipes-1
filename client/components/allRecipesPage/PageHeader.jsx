import React from 'react';


/**
 * @description displays a page title/header
 * @method PageHeader
 *
 * @returns { jsx } jsx - renders PageHeader component
 */
const PageHeader = () => (
  <section className="cover text-center mb-3">
    <h3 className="page-text">All Recipes</h3>
    <hr id="favorites"/>
  </section>
);

export default PageHeader;
