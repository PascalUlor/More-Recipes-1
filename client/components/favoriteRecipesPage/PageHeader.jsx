import React from 'react';


/**
 * @description displays a page title/header
 * @method PageHeader
 *
 * @returns { jsx } jsx - renders PageHeader component
 */
const PageHeader = () => (
  <section className="cover text-center">
    <div className="col">
    <h3 className="page-header p-2">My Favorites</h3>
    </div>
    <hr id="favorites"/>
  </section>
);

export default PageHeader;
