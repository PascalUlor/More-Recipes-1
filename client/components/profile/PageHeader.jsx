import React from 'react';


/**
 * @description displays a page title/header
 * @method PageHeader
 *
 * @returns { jsx } jsx - renders PageHeader component
 */
const PageHeader = () => (
  <section className="cover text-center">
    <div className="col">
      <h3 className="page-header p-2 mb-1">My Profile</h3>
    </div>
    <hr id="favorites"/>
  </section>
);

export default PageHeader;