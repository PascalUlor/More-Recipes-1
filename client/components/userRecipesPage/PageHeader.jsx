import React from 'react';


const PageHeader = () => (
  <section className="cover text-center">
    <div className="col">
      <h3 className="page-header p-2 mb-1">My Recipes</h3>
    </div>
    <hr id="favorites"/>
  </section>
);

export default PageHeader;