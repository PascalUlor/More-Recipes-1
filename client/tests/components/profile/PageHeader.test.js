/* eslint-disable */
import React from 'react';
import toJson from 'enzyme-to-json';
import PageHeader from '../../../components/profile/PageHeader.jsx';


describe('<Footer/>', () => {
  it('renders page header for profile component without crashing', () => {
    const shallowWrapper = shallow(<PageHeader/>);
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
