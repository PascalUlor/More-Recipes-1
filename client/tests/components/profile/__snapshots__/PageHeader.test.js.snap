// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders page header for profile component without crashing 1`] = `
<section
  className="cover text-center"
>
  <div
    className="col"
  >
    <h3
      className="page-header p-2 mb-1"
    >
      My Profile
    </h3>
  </div>
  <hr
    id="favorites"
  />
</section>
`;
